src/Statement/StatementList.php

```diff
@@ -106,6 +107,25 @@ public function addNewStatement( Snak $mainSnak, $qualifiers = null, $references
 	}
 
 	/**
+	 * @param string $guid
+	 * @return Statement[]
```

Comment:
Yep, should not do the return

https://en.wikipedia.org/wiki/Command%E2%80%93query_separation